ocs_ci/ocs/constants.py

```diff
@@ -15,6 +15,7 @@
     os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 )
 TEMPLATE_DIR = os.path.join(TOP_DIR, "ocs_ci", "templates")
+EXTERNAL_DIR = os.path.join(TOP_DIR, "external")
```

Comment:
Is the `EXTERNAL_DIR` used anywhere? I would expect, that it will be used for the _openshift-misc_ repo, but it seems to be cloned to `<cluster_path>/openshift-misc`, which is quite problematic on Jenkins (_cluster_path_ is on NFS share, and there is a problem with cloning git repository on NFS share exported by nfs-ganesha - [RHBZ#1735480](https://bugzilla.redhat.com/show_bug.cgi?id=1735480))

Reply:
That was the original purpose of the external_dir, but we decided to go with cloning into the cluster_path to keep things together. We will go back to cloning into the external dir to work around this issue in jenkins.

Reply:
We are now cloning the repo into the `external` dir, sym-linking the `install-dir` to `cluster_path`, and removing the repo after deployment is done.